libraries/aws_alb.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsAlb < AwsResourceBase
+  name 'aws_alb'
+  desc 'Verifies settings for an Application Load Balancer'
+
+  example "
+    describe aws_alb('arn:aws:elasticloadbalancing') do
+      it { should exist }
+    end
+  "
+
+  attr_reader :availability_zones, :canonical_hosted_zone_id, :created_time, :dns_name, :load_balancer_arn,
+              :load_balancer_name, :load_balancer_addresses, :scheme, :security_groups, :state, :subnets, :type, :vpc_id, :zone_names
+
+  def initialize(opts = {})
+    opts = { load_balancer_arns: opts } if opts.is_a?(String)
```

Comment:
This creates the key `load_balancer_arns` but the rest of the class seems to work with the opts key `load_balancer_arn`
(if I'm just misreading this, sorry)

Reply:
that's a good catch @james-stocks - we should have a failing unit test for the `String` case ...

Reply:
Good spot! I've added a unit test for this too.